### PR TITLE
feat(kit): new component `ProgressCircle`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [X] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [X] No

### CODE REVIEW

File: projects/kit/components/progress/progress-circle/progress-circle.style.less [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.progress {
+    fill: transparent;
+    stroke: var(--tui-progress-color, currentColor);
+    transition: stroke-dashoffset var(--tui-duration) linear;
+}
+
+.hidden-progress {
+    display: none;
```
Comment (waterplea) [inline projects/kit/components/progress/progress-circle/progress-circle.style.less:78]: This way accessibility tools will not see it as well. We need to make it transparent instead. You can install NVDA to check with screen reader, I believe it's the most popular free solution. I'm not sure opacity: 0 will be visible, need to be tested.

**Review by waterplea — CHANGES_REQUESTED** [review]

File: projects/kit/components/progress/progress-circle/progress-circle.style.less (hunk 1 above)
Comment (nsbarsukov) [reply projects/kit/components/progress/progress-circle/progress-circle.style.less]: Make changes according to this [article](https://webaim.org/techniques/css/invisiblecontent/).

tailwindcss [does it in the same way](https://tailwindcss.com/docs/screen-readers).

File: projects/kit/components/progress/progress-circle/progress-circle.style.less [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        .circle-params(xl);
+    }
+}
+
+.track {
+    fill: transparent;
+    stroke: var(--tui-base-03);
+}
+
+.progress {
```
Comment (nsbarsukov) [inline projects/kit/components/progress/progress-circle/progress-circle.style.less:73]: need to add `t-` prefix for all new classes in progress-circle

**Review by waterplea — APPROVED** [review]

**Comment by nsbarsukov** [comment]
@waterplea 
I made small changes in styles.
Can I merge PR ?

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**